tools/core/Pims.Tools.Core.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.9" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.9" />
     <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="3.1.9" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.8.0" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.11.0" />
```

Comment:
I thought these were already completed.  I don't understand why your branch is showing that it will be changed again?

Reply:
yeah, got conflict with this PR and accepted your earlier changes...